waspc/examples/todoApp/ext/Main.css

```diff
@@ -1,64 +1,42 @@
-div {
-    color: green;
-}
-
-button.selected {
-    border: 1px solid black;
-}
-
-.hidden {
-    visibility: hidden;
-}
-
-.todos {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-}
-
-.todos h1 {
-    text-align: center;
-}
-
-.todos__container {
-    width: 450px;
-}
-
-.todos__toggleAndInput {
-    display: flex;
-    align-items: flex-end;
-    margin-bottom: 10px;
-}
-
-.todos__toggleButton {
-    height: 40px;
-    margin-bottom: 8px;
-    margin-left: 20px;
-
-    background-color: white;
-    border: none;
-    display: inline-block;
-    font-size: 24px !important;
-    cursor: pointer;
-}
-
-.todos__newTaskForm {
-    width: 100%;
-    margin-left: 10px;
-}
-
-.todos__newTaskForm__input {
-  margin: 0 26px !important;
-}
-
-.todos__footer {
-    width: 100%;
-    margin-top: 10px;
-
-    display: flex;
-    justify-content: space-between;
-}
-
-.todos__footer__filters .filter:not(:last-child) {
-    margin-right: 10px;
+@tailwind base;
```

Comment:
I'm not 100% sure on the tailwind/postcss config stuff, since I just took the defaults that were spit out from this guide: https://tailwindcss.com/docs/guides/create-react-app So take all answers here with a grain of salt :D 

However, it sounds like `@tailwind` stuff is a custom CSS "at-rule". It appears at-rules are extensible, vanilla CSS statements that instruct it how to behave.

`@apply` and `@layer` I believe are used by PostCSS (as is the Tailwind directive). https://tailwindcss.com/docs/using-with-preprocessors

Reply:
Interesting, thanks, I will have to read more about this!